### ISSUE TITLE
Document the ability to run local generators

### ISSUE THREAD

**Issue by mischah** [yeoman/yo#546]
### Documentation improvement

The ability to run local generators with `yo ./path/to/some/generator` should be documented in:
+ the README
+ the CLIs help output

----

This popped up via #545

**Comment by buoyantair** [comment]
Where is the CLIs help output declared?

**Comment by mischah** [comment]
It’s in this:
https://github.com/yeoman/yo/blob/master/lib/usage.txt#L11

Would prefer an own section like:

```
Install a generator:

  Generators can be installed through npm.

  $ npm install generator-angular
  $ yo angular --help

Run local generators:

  You can also run local generators without installing via npm.

  $ yo ./path/to/some/generator

```

**Comment by buoyantair** [comment]
Ohh @mischah Thank you, I didn't realize that.

**Comment by mischah** [comment]
Fixed with #547 ✨
